interface/portraits/portraits_naga.gfx

```diff
@@ -11,36 +11,13 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
 			"GFX_character_background:p0"
-			"GFX_empty:p3"
-			"GFX_empty:p5"
-			"GFX_empty:p1:h:y"
-			"GFX_empty:p4:h:y"
-			":p5:c6"
 			
 			"GFX_naga_male_base:p2"
 			"GFX_naga_male_neck:d0"
```

Comment:
The portraits look and work fine except for one thing that can be fixed.
In the Ruler Designer, it shows a few options for facial features despite having none.
The eyes shows 13 options, but there's no eye varitation.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/100639327-75197380-334e-11eb-9b3d-d9269f857ead.png)

</details>

It can be fixed by having sprites for these facial features. The sprites must reference `empty.dds` and be named with the corresponding facial feature (`neck`, `chin`, etc.)

I fixed it for the trolls not so long ago. So it's your example.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/100639819-1a344c00-334f-11eb-8a54-c849c2e53739.png)

</details>

`GFX_jungletroll_male_mouth_behind` is solely a troll sprite and used by tusks. Don't add it for the naga.